packaging_automation/update_package_properties.py

```diff
@@ -278,6 +283,7 @@ def validate_package_properties_params_for_update_all_changes(package_props: Pac
 
 
 @validate_parameters
+# pylint: disable=no-value-for-parameter
```

Comment:
maybe comment the reasoning for that ignore rule here, as we did in others

Reply:
Fixed